### PR TITLE
added palindrome checker code in java

### PR DESCRIPTION
**Pull request by dragon540**
**added palindrome checker code for java**

### CODE REVIEW

File: code/utility/src/palindrome/palindrome_check/Palindrome.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* checks whether a given string is palindrome or not */
```
Comment (arnavb) [inline code/utility/src/palindrome/palindrome_check/Palindrome.java]: The file name should be lowercase and called palindrome_check.java.

**Comment by arnavb** [comment]
@dragon540 Please delete the older file!

**Comment by dragon540** [comment]
deleted that file.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**